cartography/intel/aws/route53.py

```diff
@@ -114,6 +115,70 @@ def load_zone(neo4j_session, zone, current_aws_id, update_tag):
     )
 
 
+def load_ns_records(neo4j_session, records, zone_name, update_tag):
+    ingest_records = """
+    UNWIND {records} as record
+    MERGE (a:DNSRecord:AWSDNSRecord{id: record.id})
+    ON CREATE SET a.firstseen = timestamp(), a.name = record.name, a.type = record.type
+    SET a.lastupdated = {aws_update_tag}, a.value = record.name
+    WITH a,record
+    MATCH (zone:AWSDNSZone{zoneid: record.zoneid})
+    MERGE (a)-[r:MEMBER_OF_DNS_ZONE]->(zone)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    WITH a,record
+    UNWIND record.servers as server
+    MERGE (ns:NameServer{id:server})
+    ON CREATE SET ns.firstseen = timestamp()
+    SET ns.lastupdated = {aws_update_tag}, ns.name = server
+    MERGE (a)-[pt:DNS_POINTS_TO]->(ns)
+    SET pt.lastupdated = {aws_update_tag}
+
+    """
+    neo4j_session.run(
+        ingest_records,
+        records=records,
+        aws_update_tag=update_tag,
+    )
+
+    # Map the official name servers for a domain.
+    map_ns_records = """
+    UNWIND {servers} as server
+    MATCH (ns:NameServer{id:server})
+    MATCH (zone:AWSDNSZone{zoneid:{zoneid}})
+    MERGE (ns)<-[r:NAMESERVER]-(zone)
+    SET r.lastupdated = {aws_update_tag}
+    """
+    for record in records:
+        if zone_name == record["name"]:
+            neo4j_session.run(
+                map_ns_records,
+                servers=record["servers"],
+                zoneid=record["zoneid"],
+                aws_update_tag=update_tag,
+            )
+
+
+def link_sub_zones(neo4j_session, update_tag):
+    query = """
+    match (z:AWSDNSZone)
+    <-[:MEMBER_OF_DNS_ZONE]-
+    (record:DNSRecord{type:"NS"})
+    -[:DNS_POINTS_TO]->
+    (ns:NameServer)
+    <-[:NAMESERVER]-
+    (z2)
+    WHERE record.name=z2.name AND NOT z=z2
```

Comment:
For not equal, what do you think about`<>` instead of `NOT`?  I won't block on this comment but I think `<>` is easier to read but that could be just me.